pom.xml

```diff
@@ -44,6 +44,10 @@
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <gitRepositoryName>sonarlint-cli</gitRepositoryName>
     <sonarlint.version>2.5.0.36</sonarlint.version>
+    <sonar.java.version>4.2.1.6971</sonar.java.version>
+    <sonar.javascript.version>2.18.0.3454</sonar.javascript.version>
+    <sonar.php.version>2.9.0.1664</sonar.php.version>
+    <sonar.python.version>1.6</sonar.python.version>
```

Comment:
Need 2 tickets: 1 for addition of Python, and another for the update of all analyzers.